types/testing-library__dom/testing-library__dom-tests.ts

```diff
@@ -19,3 +19,8 @@ function testByRole() {
     console.assert(queryByRole(element, 'button') === null);
     console.assert(queryByRole(element, 'button', { hidden: true }) !== null);
 }
+
+function testA11yHelper() {
+    const element = document.createElement('svg');
+    console.assert(isInaccessible(element) === false);
```

Comment:
```suggestion
    console.assert(!isInaccessible(element));
```

Fixing PRs from my phone. What a time to be alive.